assets/js/components/settings/SettingsActiveModule/Header.test.js

```diff
@@ -38,6 +38,7 @@ import {
 	fireEvent,
 	waitFor,
 	provideUserInfo,
+	provideModuleRegistrations,
```

Comment:
Nice, this definitely simplifies stuff, doesn't it. Not sure I completely forgot about it since I used it a couple of months ago in another test. :facepalm: